tests/phpunit/tests/functions.php

```diff
@@ -1225,6 +1225,29 @@ public function test_wp_get_image_mime( $file, $expected ) {
 		$this->assertSame( $expected, wp_get_image_mime( $file ) );
 	}
 
+	/**
+	 * @ticket 35725
+	 * @dataProvider _wp_getimagesize
```

Comment:
Nit-pick but the common best practice here is to name the data provider method exactly the same as the test method, with `test_` replaced with `data_`, so this would be more appropriate as `data_wp_getimagesize`.

Reply:
good point, adjusted.